forum_service.py

```diff
@@ -1,40 +1,51 @@
 from rs_scraper import RSScraper
+from post_cleaner import PostCleaner
+from datetime import datetime as dt
 
 class ForumService:
     """
-        Handles the scraper.
+        Handles the scraping calls and cleaning data
     """
-    def __init__(self):
-        pass
-
-    ##################
-    # Scraping Methods
-    ##################
-    def get_raw_posts(self, url, page_start, page_end=''):
-        scraper = RSScraper(url)
-        if (page_end == '') : page_end = scraper.max_page #defaults to last page if a specific page is not specified
-        raw_posts = []
-        for i in range(page_start, page_end):
-            raw_posts += scraper.scrape_page(i)
-        return raw_posts, scraper.current_page, scraper.current_post
-        
-
-
 
-    def get_all_price_posts(self, url):
+    def get_clean_forum_posts(self, url, thread_id, page_start, page_end=''):
```

Comment:
I think we can just name this `get_forum_posts(...)` because it should be clear that you'd never want forum posts that couldn't be parsed anywhere else
```suggestion
    def get_forum_posts(self, url, thread_id, page_start, page_end=''):
```